### PR TITLE
The code for my "Week 4 Goals" issue.

### LINKED ISSUE #21: [Week 4 Goals]
* Create map for layout of rooms
* Add details and flow to combat to make it more interesting
* Work on and edit Story class

### PR DESCRIPTION
**Pull request by TMGWill**

### CODE REVIEW

File: 486_Project/Characters_Class.py [hunk 1]
```diff
@@ -46,8 +52,12 @@ def __init__(self):
     def new_max_hp(self):
         self.warrior_max_hp = self.stats[3] * 4
 
+    # function for when player gets new weapon
+    def new_weapon(self):
+        pass
+
 
-class bad_warrior:
+class Bad_warrior:
```
Comment (jazzij) [inline 486_Project/Characters_Class.py:60]: Is this a sub-class of Warrior, inheriting all the attributes and adding a few extra?

**Comment by jazzij** [comment]
This Pull request seems to have a lot of things in progress. Would like to see next week 1) A Map with all Rooms, 2) A Room init with all rooms intro test and decorators,  3) All Classes of characters and enemies implemented. This would put you in a better place to actually finish your game in time to test it

File: 486_Project/Text_Game.py [hunk 2]
```diff
@@ -5,20 +5,38 @@
...
 # story intro
 story.intro()
 
 # enemies
-bw1 = bad_warrior()
-bw2 = bad_warrior()
-w1 = wolf()
+bw1 = Bad_warrior()
+bw2 = Bad_warrior()
+w1 = Wolf()
```
Comment (walkerje) [inline 486_Project/Text_Game.py:39]: Could these exist as part of your room's state? ```myRoomVar.enemy = Bad_warrior()```, for example.

**Review by walkerje — APPROVED** [review]
Great work! I agree with Dr. Jones; I see a lot in progress here all at once. The steps that were taken towards making your code generic are great! Feel free to accept this PR.

**Pull request merged**